app/src/main/java/org/oppia/android/app/player/stopplaying/StopStatePlayingSessionWithSavedProgressListener.kt

```diff
@@ -0,0 +1,13 @@
+package org.oppia.android.app.player.stopplaying
+
+/** Listener for when the current state playing session should be stop and the user navigated back to the topic. */
+interface StopStatePlayingSessionWithSavedProgressListener {
+  /** leave the currently playing session without saving the current progress. */
+  fun deleteCurrentProgressAndStopSession()
+
+  /**
+   * leave the currently playing session and overwrite the oldest saved progress with the current
```

Comment:
The verb tense & capitalization is wrong here & above--please update to be consistent with the broader codebase & our conventions.

Reply:
done